base/shell/cmd/lang/id-ID.rc

```diff
@@ -100,9 +100,9 @@ CTTY device\n\n\
             CON is usually the default standard I/O device.\n\n\
 To return control to the standard console, type: CTTY CON on the auxiliary\n\
 device."
-    STRING_DATE_HELP1 "\nMasukkan tanggal baru (mm%cdd%cyyyy): "
-    STRING_DATE_HELP2 "\nMasukkan tanggal baru (dd%cmm%cyyyy): "
-    STRING_DATE_HELP3 "\nMasukkan tanggal baru (yyyy%cmm%cdd): "
+    STRING_DATE_HELP1 "\nMasukkan tanggal baru (format: mm%cdd%cyyyy): "
```

Comment:
preferred to keep this. (regarding to other language resource format)
```suggestion
    STRING_DATE_HELP1 "\nMasukkan tanggal baru (mm%cdd%cyyyy): "
```
including two following line below

Reply:
Reverted in https://github.com/reactos/reactos/pull/3417/commits/60f67146b89693a5d391a9481a30f7965dae4df5